cardano-cli/test/Test/CLI/TextEnvelope/Golden/Genesis/Create.hs

```diff
@@ -0,0 +1,126 @@
+{-# LANGUAGE OverloadedStrings    #-}
```

Comment:
Can we move `Create.hs` to `cardano-cli/test/Test/CLI/Shelley/Golden/Genesis/Create.hs`?

Reply:
Sure